firefly/main.py

```diff
@@ -8,15 +8,25 @@
 from wsgiref.simple_server import make_server
 
 def load_from_env():
+    functions = None
+    token = None
     if 'FIREFLY_FUNCTIONS' in os.environ:
         function_names = os.environ['FIREFLY_FUNCTIONS'].split(",")
         try:
             functions = load_functions(function_names)
         except (ImportError, AttributeError) as err:
             sys.exit(1)
-        add_routes(app, functions)
+
     if 'FIREFLY_TOKEN' in os.environ:
         token = os.environ['FIREFLY_TOKEN']
+
+    if 'FIREFLY_CONFIG' in os.environ:
+        functions, token = parse_config_data(parse_config_file(os.environ['FIREFLY_CONFIG']))
```

Comment:
The API as such right now has many loopholes and may become complicated. I am working on streamlining that. Will discuss with you separately on that.